atomics/T1176/T1176.yaml

```diff
@@ -64,3 +64,40 @@ atomic_tests:
       2. Click 'Get'
     name: manual
 
+- name: Google Chrome Load Unpacked Extension With Command Line
+
+  description: |-
+    This test loads an unpacked extension in Google Chrome with the `--load-extension` parameter. This technique was previously used by the Grandoreiro malware to load a malicious extension that would capture the browsing history, steal cookies and other user information. Other malwares also leverage this technique to hijack searches, steal passwords, inject ads, and more.
+    
+    References:
+    https://attack.mitre.org/techniques/T1176/
+    https://securityintelligence.com/posts/grandoreiro-malware-now-targeting-banks-in-spain/
+
+  supported_platforms:
+  - windows
+  executor:
+    command: |-
+
+      # Chromium
+      $chromium =  "https://commondatastorage.googleapis.com/chromium-browser-snapshots/Win_x64/1153778/chrome-win.zip"
+
+      # uBlock Origin Lite to test side-loading
+      $extension = "https://github.com/gorhill/uBlock/releases/download/uBOLite_0.1.23.6055/uBOLite_0.1.23.6055.chromium.mv3.zip"
+
+      Set-Location $env:TEMP
+
+      Invoke-WebRequest -URI $chromium -OutFile $env:TEMP\chrome.zip
+      Invoke-WebRequest -URI $extension -OutFile $env:TEMP\extension.zip
+
```

Comment:
It takes a lot of time to download the files using `Invoke-WebRequest`. if I change the `ProgressPreference` to `SilentlyContinue`, it downloads faster. Can you make a change to do that and after downloading you can reset the `ProgressPreference` to the previous value ?

Reply:
Yup